### PR TITLE
feat: added UI of video library

### PR DESCRIPTION
**Pull request by paarthbajaj**
homepage

### CODE REVIEW

**Review by rkapoor10 — APPROVED** [review]
- the code is well structured
- UI looks great
- CSS is well ordered
- looks good to merge

File: src/components/History.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { VideoBlock } from "./VideoBlock";
+import { Sidebar } from "./Sidebar";
+import "./VideoList.css";
+export const History = () => {
+  return (
+    <>
```
Comment (JantuDeb) [inline src/components/History.js:6]: You can remove the <></>

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { Link } from "react-router-dom";
+import "./Navbar.css";
+export const Navbar = () => {
+  return (
+    <>
```
Comment (JantuDeb) [inline src/components/Navbar.js:5]: same here

File: src/components/Sidebar.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { Link } from "react-router-dom";
+import "./Sidebar.css";
+export const Sidebar = () => {
+  return (
+    <>
```
Comment (JantuDeb) [inline src/components/Sidebar.js:5]: here also

**Review by JantuDeb — APPROVED** [review]
This is looking fine

**Pull request merged**